ACTIVITYLOG.md

```diff
@@ -0,0 +1,11 @@
+# Activity Log
+
+If you are making a low or no-code contribution to this project that does NOT involve making a pull request, please log your activity here: 
```

Comment:
they will need to do a pull request to edit this file to include what they have done. As hacktoberfest tracks pull requests